### ISSUE TITLE
Bug in spec file?

### ISSUE THREAD

**Issue by EricSchwartz7** [learn-co-curriculum/my-collect#20]
Hi Flatiron,

Lines 40 and 41 of my_collect_spec.rb read:

```
expect(my_collect(students) do |language|
  student.split(" ").first
```

I think it should be:

```
expect(my_collect(students) do |student|
  student.split(" ").first
```

I changed it and it got the test to pass.  I hope I didn't just cheat my way through the lesson. haha

**Comment by emilycroft** [comment]
I had the same experience, as soon as I saw the error I came to see if anyone else had reported this issue. Also hoping this isn't cheating--

**Comment by gj** [comment]
Fixed in #17. Thanks @EricSchwartz7 !
